src/System.Management.Automation/CoreCLR/CorePsStub.cs

```diff
@@ -689,11 +689,12 @@ public void Dispose()
     #endregion
 }
 
+#if UNIX
+
 namespace System.Management.Automation.Security
 {
     /// <summary>
-    /// We don't need Windows Lockdown Policy related types on CSS because CSS is
-    /// amd64 only and is used internally.
+    /// Application white listing security policies only affect Windows OSs
```

Comment:
Please add final dot in the _public_ comment.

Reply:
Will do.